packages/foreman/rubygem-unf/rubygem-unf.spec

```diff
@@ -1,83 +1,74 @@
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name unf
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.1.3
-Release: 9%{?dist}
-Summary: Unicode normalization form support for Ruby/JRuby
-Group: Development/Languages
-License: MIT
+Name: rubygem-%{gem_name}
+Version: 0.1.4
+Release: 1%{?dist}
+Summary: A wrapper library to bring Unicode Normalization Form support to Ruby/JRuby
```

Comment:
The old description was better IMHO. A bit shorter. Should we submit a PR upstream to change it while changing it locally?

Reply:
I think submitting a PR to upstream is fine but I do not feel we should be changing it here. This is the project's summary and even though we may disagree with it in some way we should keep it as is from the source. That also prevents it getting changed in the future every time and having to try and recall why.

Reply:
Upstream hasn't seen a commit in the past 5 years.